skillsmap/src/lib/skillMapParser.ts

```diff
@@ -41,6 +42,40 @@ type: tutorial
 
 url: /tutorials/barrel-dodger
 imageUrl: /static/lessons/barrel-dodger.png
+
+# map2
+name: test skills map2
```

Comment:
I've tested this in crowdin and it treats the 3 lines as a single paragraph. I think you'll want bullet points.

![image](https://user-images.githubusercontent.com/4175913/97572278-5158bb80-19a5-11eb-87d2-390699436ca7.png)